### PR TITLE
Security component

### PR DESCRIPTION
**Pull request by EVDW**

### CODE REVIEW

File: _includes/_components/fixtures-load.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <header>
         <div class="container">
             <h2>Fixtures Load</h2>
-            <p>Allows you to use nelmio/alice library when loading your fixtures.</p>
+            <p>Allows you to use <a href="https://github.com/nelmio/alice">nelmio/alice</a> library when loading your fixtures.</p>
```
Comment (akovalyov) [inline _includes/_components/fixtures-load.html]: `target="_blank"` ?

File: _includes/_components/security.html [hunk 2]
```diff
@@ -1,50 +1,90 @@
...
+namespace App\Controller;
+
+class ArticleController extends Controller
 {
-    public function test(SomeClass $subject)
+    public function indexAction()
     {
-        return $subject->doSomething();
+        if (!$this->isGranted('IS_AUTHENTICATED_FULLY')) {
+            throw $this->createUnauthorizedException();
```
Comment (akovalyov) [inline _includes/_components/security.html]: `$this->denyUnlessGranted` can be used here.

File: _layouts/default.html [hunk 3]
```diff
@@ -28,15 +28,15 @@
...
       {% include _components/auto-registration.html %}
-      {% include _components/user.html %}
       {% include _components/fixtures-load.html %}
-      {% include _components/security.html %}
+      {% include _components/user.html %}
       {% include _components/view-renderer.html %}
       {% include _components/domain-event.html %}
       {% include _components/url-generation.html %}
-      {% include _components/prototype.html %}
+      <!--{% include _components/prototype.html %}-->
```
Comment (akovalyov) [inline _layouts/default.html]: I guess, twig comment would be more suitable here. As far as I understand, the template is still being included.

Comment (EVDW) [inline _layouts/default.html]: It's not twig, it's jekyll! And I I didn't find comments tag in jekyll:/

File: _includes/_components/security.html [hunk 4]
```diff
@@ -1,50 +1,88 @@
...
-        <div class="container content">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
-          <div class="row">
+<article class="component dark-blue">
+    <header>
+        <div class="container">
+        <h2>Security</h2>
+        <p>Establish a security at the routing and not at the controller.</p>
```
Comment (AntoineLelaisant) [inline _includes/_components/security.html]:
``` html
<p>Set up security check from the routing instead of the controller.</p>
```

**Pull request merged**